scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py

```diff
@@ -230,43 +256,48 @@ def populate_final_extract_table(fq_temp_table_dataset, fq_destination_dataset,
   cohort_extract_final_query_job = client.query(sql, job_config=job_config)
 
   cohort_extract_final_query_job.result()
-  JOB_IDS.add((f"insert final cohort table {dest}", cohort_extract_final_query_job.job_id))
+  JOB_IDS.add((f"insert final cohort table {fq_destination_table_data}", cohort_extract_final_query_job.job_id))
   return
 
 def make_extract_table(fq_pet_vet_dataset,
                max_tables,
+               sample_names_to_extract,
                fq_cohort_sample_names,
                query_project,
                query_labels,
                fq_temp_table_dataset,
                fq_destination_dataset,
-               destination_table,
+               destination_table_prefix,
                min_variant_samples,
                fq_sample_mapping_table,
                sa_key_path,
                temp_table_ttl_hours
               ):
   try:
+    fq_destination_table_data = f"{fq_destination_dataset}.{destination_table_prefix}__DATA"
+    fq_destination_table_samples = f"{fq_destination_dataset}.{destination_table_prefix}__SAMPLES"
+    
     global client
     # this is where a set of labels are being created for the cohort extract
     query_labels_map = {}
-    query_labels_map["id"]= {output_table_prefix}
-    query_labels_map["gvs_tool_name"]= f"gvs_prepare_callset"
+    query_labels_map["id"]= output_table_prefix
+    query_labels_map["gvs_tool_name"]= "gvs_prepare_callset"
+
     # query_labels is string that looks like 'key1=val1, key2=val2'
-    if len(query_labels) != 0:
+    if query_labels is not None and len(query_labels) != 0:
         for query_label in query_labels:
           kv = query_label.split("=", 2)
           key = kv[0].strip().lower()
           value = kv[1].strip().lower()
           query_labels_map[key] = value
 
-    if not (bool(re.match(r"[a-z0-9_-]+$", key)) & bool(re.match(r"[a-z0-9_-]+$", value))):
-      raise ValueError(f"label key or value did not pass validation--format should be 'key1=val1, key2=val2'")
-
+          if not (bool(re.match(r"[a-z0-9_-]+$", key)) & bool(re.match(r"[a-z0-9_-]+$", value))):
+            raise ValueError(f"label key or value did not pass validation--format should be 'key1=val1, key2=val2'")
+        
```

Comment:
I want to add testing for all of this, but am procrastinating it because it seems like this will become Java at some point soon, but maybe that's not true....